include/aoapplication.h

```diff
@@ -596,6 +406,9 @@ class AOApplication : public QApplication {
   QHash<uint, QString> dir_listing_cache;
   QSet<uint> dir_listing_exist_cache;
 
+  Options* options = nullptr;
+  AOOptionsDialog* settings_widget = nullptr;
```

Comment:
```suggestion
  AOOptionsDialog *settings_widget;
```
you don't need options because it's a singleton. you don't need to declare a pointer as nullptr; that is more or less implicit